restd/pom.xml

```diff
@@ -121,6 +121,12 @@
             <version>4.11</version>
             <scope>test</scope>
         </dependency>
+        <!-- TODO Investigate if this dependency is required -->
```

Comment:
I investigated that. And yes, this new dependency is required. However, this did not really cost much time since this regression was simple to fix.